config/database.js

```diff
@@ -0,0 +1,12 @@
+require('dotenv').config();
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize({
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  host: process.env.DB_HOST,
+  dialect: process.env.DB_DIALECT,
+});
```

Comment:
`Suggestion`: Instead of writing `process.env` again and again we can destructure `env` from the process or we can `destructure` all the keys as well. Also, we can refactor this code as well.